the-badass-todo-list-part-1/input.js

```diff
@@ -18,16 +20,20 @@
         };
         // To prevent user saving multiple times.
         input.disabled = true;
+        // Quiz #2
         saveData(data, tidyUp);
       }
     });
   }
 
+  // Quiz #2
   function saveData(todoItem, afterSave) {
     console.log('saved');
+    // Quiz #2
     afterSave();
   }
 
+  // Quiz #2
   function fetchData(afterFetch) {
     var xhr = new XMLHttpRequest();
     xhr.open('GET', 'http://127.0.0.1:8000/demo-input-motd.json', true);
```

Comment:
Good point. In the real world, we usually have some config files to put things like this.